flowey/flowey_hvlite/src/pipelines/build_igvm.rs

```diff
@@ -153,6 +153,10 @@ pub struct BuildIgvmCliCustomizations {
     #[clap(long)]
     pub custom_directory: Vec<PathBuf>,
 
+    /// Additional rootfs.config files to use to generate the initrd
+    #[clap(long)]
+    pub additional_rootfs: Vec<PathBuf>,
```

Comment:
can you stick with using the `custom_` prefix, for consistency? e.g: `custom_extra_rootfs`